tools/loader/Loader.cpp

```diff
@@ -419,7 +419,7 @@ void Loader::compile(CompilationContext &cctx) {
                    mainEntryName.empty() ? networkName : mainEntryName);
   } else {
     // Emit IR for the graph and compile it.
-    auto error = hostManager_->addNetwork(std::move(M_), cctx);
+    auto error = hostManager_->addNetwork(std::move(M_), cctx, true);
```

Comment:
Do we want to add a command line option for this, instead of hard coding true?

Reply:
I think hardcoding here is fine, Since image-classifier only loads one network I think replication is the only real use when num-devices>1.

Reply:
What about partitioning? I guess if we need to use partitioning it just won't clone any of the partitions functions to the devices? Actually, curious what happens if we have 3 devices, 2 partitions, and saturate is true...?

Reply:
Then we don't duplicate, saturate only happens if there is enough space for a full copy of the network on available devices.